content/references/rippled-api/commandline-usage.md

```diff
@@ -0,0 +1,166 @@
+| `--load`              | **DEPRECATED** Intended for debugging. Only load the initial ledger from the ledger store on disk. |
+| `--replay`            | Intended for debugging. Use with `--ledger` to replay a ledger close. Your server must have the ledger in question and its direct ancestor already in the ledger store. Using the previous ledger as a base, the server processes all the transactions in the specified ledger, resulting in a re-creation of the specified ledger. With a debugger, you can add breakpoints to analyze specific transaction processing logic. |
+| `--start`             | Intended for debugging. Start with a new genesis ledger that has all known amendments (except those the server is configured to vote against) enabled. The functionality of those amendments is therefore available starting from the second ledger, rather than going through the full two-week [Amendment Process](amendments.html). |
+| `---valid`            | **DEPRECATED** Intended for debugging. Consider the initial ledger a valid network ledger even before fully syncing with the network. |
+
+## Client Mode Options
+
+```bash
+rippled [OPTIONS] -- {COMMAND} {COMMAND_PARAMETERS}
+```
+
+In client mode, the `rippled` executable acts as a client to another `rippled` service. (The service may be the same executable running in a separate process locally, or it could be a `rippled` server on another server.)
+
+To run in client mode, provide the [commandline syntax](request-formatting.html#commandline-format) for one of the [`rippled` API](rippled-api.html) methods.
+
+In addition to the individual command syntax, client mode accepts the [Generic Options](#generic-options) and the following options:
+
+| Option                  | Description                                        |
+|:------------------------|:---------------------------------------------------|
+| `--rpc`                 | Explicitly specify that the server should run in client mode. Not required. |
+| `--rpc_ip {IP_ADDRESS}` | Connect to the `rippled` server at the specified IP Address, optionally including a port number. |
+| `--rpc_port {PORT}`     | **DEPRECATED** Connect to the `rippled` server on the specified port. Specify the port alongside the IP address using `--rpc_ip` instead. |
+
+**Tip:** Some arguments accept negative numbers as values. To ensure that arguments to API commands are not interpreted as options instead, pass the `--` argument before the command name.
+
+Example usage (get account transaction history from the earliest available to latest available ledger versions):
+
+```bash
+rippled -- account_tx r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 -1 -1
+```
+
+
+## Unit Tests
+
+```bash
+rippled --unittest [OPTIONS]
+rippled -u [OPTIONS]
+```
+
+Unit testing runs tests built into the `rippled` source code to confirm that the executable performs as expected. This includes things like testing built-in data types and transaction processing routines.
```

Comment:
Is it worth saying explicitly that this option runs the specified unit tests, reports a summary, and then quits?  Your call.